Source/GUI/MemWatcher/MemWatchWidget.cpp

```diff
@@ -393,6 +430,14 @@ void MemWatchWidget::pasteWatchFromClipBoard(const QModelIndex& referenceIndex)
   m_hasUnsavedChanges = true;
 }
 
+void MemWatchWidget::copyAddressToClipboard(u32 addr)
```

Comment:
Nit: `addr` should be marked as `const`.

Reply:
This method is no longer present, see above